src/cursor.js

```diff
@@ -3,78 +3,86 @@ import convertToNative from './convert_to_native';
 import Immutable from 'immutable';
 import putOnChan from './put_on_chan';
 
-const hasSameValue = function (valueA, cursorB) {
+function hasSameValue(valueA, cursorB) {
   return isEqual(valueA, Immutable.fromJS(cursorB.deref()));
-};
+}
 
-const isEqual = function (valueA, valueB) {
+function isEqual(valueA, valueB) {
```

Comment:
Should look into that, though I don't want to pull in ramda in StateTrooper atm.